packages/atomic/src/components/common/button.spec.ts

```diff
@@ -1,70 +1,60 @@
 import {createRipple} from '@/src/utils/ripple';
-import {fireEvent, within} from '@storybook/test';
-import {html, nothing, render} from 'lit';
-import {vi} from 'vitest';
-import {button, ButtonProps} from './button';
+import {fixture} from '@/vitest-utils/testing-helpers/fixture.js';
+import {userEvent} from '@vitest/browser/context';
+import {html, nothing} from 'lit';
+import {vi, expect, describe, it} from 'vitest';
+import {button, ButtonProps} from './button.js';
 
-vi.mock('@/src/utils/ripple', () => ({
-  createRipple: vi.fn(),
-}));
+vi.mock('@/src/utils/ripple', {spy: true});
 
 describe('button', () => {
```

Comment:
I think Atom/Functional Component MAY have before/after each/all hooks, but SHOULD avoid them.

The rationale is that Atoms are meant to be "pure/stateless", so the Arrange/Setup phase does not really have a reason to be, and those hooks are often used for that.

Reply:
agreed